common/djangoapps/student/auth.py

```diff
@@ -20,7 +20,7 @@
 # In addition to the above, one is always allowed to "demote" oneself to a lower role within a course, or remove oneself
 
 
-def has_access(user, role):
+def user_has_role(user, role):
```

Comment:
Could this function name be more descriptive? `has_access_to_role` or `user_has_access_to_role`?